.github/workflows/release.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           pip install check-wheel-contents     
           check-wheel-contents dist/*.whl
       - name: Upload Artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
Out of curiosity: Are the updates of the actions necessary or just done since we cange stuff here anyway?

Reply:
n ot necessary but if you look into the actions logs it warns that some stuff is deprecated now and hence I upgraded (also some time ago in the other actions as well)